pyodide-build/pyodide_build/out_of_tree/venv.py

```diff
@@ -192,11 +181,15 @@ def create_pyodide_script(venv_bin: Path) -> None:
     environment = " ".join(environment_vars)
 
     pyodide_path = venv_bin / "pyodide"
+    original_pyodide_cli = shutil.which("pyodide")
+    if original_pyodide_cli is None:
+        raise RuntimeError("ERROR: pyodide cli not found")
+
     pyodide_path.write_text(
         dedent(
             f"""
             #!/bin/sh
-            {environment} exec {sys.executable} -m pyodide_build.out_of_tree "$@"
+            {environment} exec {original_pyodide_cli} "$@"
```

Comment:
Actually can't you use the previous apporach that only relied on PYTHONPATH? That would be more robust as compared to hoping that that pyodide is both in PATH and PYTHONPATH.

Reply:
I am not sure I am doing the correct thing here. I hope Hood can check and fix this...

Reply:
Yeah I'll try to get to it this evening. Today is the first day of classes so I've been a bit busy getting everything set up.

Reply:
Thanks!

Reply:
Well what you had worked but we can do it much simpler. To be clear, this simpler approach would have worked from the beginning but for some reason I didn't think of it the first time.